test/fuzz_group.cpp

```diff
@@ -573,6 +693,34 @@ void parse_and_apply_instructions(std::string& in, const std::string& path, util
                     t->remove(row_ndx);
                 }
             }
+            else if (instr == MERGE_ROWS && g.size() > 0) {
+                size_t table_ndx = get_next(s) % g.size();
+                TableRef t = g.get_table(table_ndx);
+                if (t->size() > 1) {
+                    size_t row_ndx1 = get_next(s) % t->size();
+                    size_t row_ndx2 = get_next(s) % t->size();
+                    if (row_ndx1 == row_ndx2) {
+                        row_ndx2 = (row_ndx2 + 1) % t->size();
+                    }
+                    // A restriction of merge_rows is that any linklists in the
+                    // "to" row must be empty because merging lists is not defined.
+                    for (size_t col_ndx = 0; col_ndx != t->get_column_count(); ++col_ndx) {
```

Comment:
IIRC, the row isn't necessarily *really* empty, but any modifications to it would have been redirected to the winning row by the merge algorithm. I would leave this as is for now, because all of this is subject to change soon anyway.